utils/isValidAirdropFile.ts

```diff
@@ -41,6 +41,9 @@ const isValidAirdropFile = (file: AirdropFileProps) => {
     toast.error('Start type must be timestamp or height or null')
     return false
   }
+  if (file.startType !== 'timestamp') {
```

Comment:
We already check for the value of startType in the next if

What we need to do is if the `start` and `expiration` field is a **string**, we need to display an error

Reply:
Still we need to get rid of this if statement because it's not used